inst/rmarkdown/templates/rjournal_article/resources/template.tex

```diff
@@ -18,10 +18,19 @@
 
 $for(author)$
```

Comment:
I reran everything and I think there is too many newlines added. There should not be one before the url. Only between the two authors.

Now: 
![image](https://user-images.githubusercontent.com/6791940/89766489-82d58f80-daf8-11ea-90ca-062edb046031.png)

Before : 
![image](https://user-images.githubusercontent.com/6791940/89766527-95e85f80-daf8-11ea-8ad8-e3e00e2bfa50.png)

Reply:
@cderv Hm, I know. This was intentional. Otherwise, it does not really format well with two affiliations. 

The idea: Two affiliations (for instance) the author may have to add because it is a request. However, it does not make much sense to have two different e-mail addresses listed (or at least it is not a requirement usually coming from the affiliation, as long as they are identified). So I added the additional line to make clear that it is valid for the two affiliations (same for ORCID and URL). Does it make somehow sense to you?

Reply:
I am just not sure it is ok with the R Journal requirements - I don't want to suggest it is if we are not sure.

I tried to find examples in the last R Journal issue, but it seems each author has only one affiliation (sometimes several departments but same adress)

Do you have an example of R Journal that would let us believe this is ok ?

Reply:
@cderv You are right, I did not find any example for these in published manuscripts either (I searched before I implemented this). However, as written initially, as an author with more than one affiliation, you do not have much of a choice. Of course, the journal could say "No we don't like this", but at the same time, they know that the pressure in academia regarding the reporting of results and the correct association of affiliation is very high. Means, I am certain that a manuscript will not become rejected because the author has to add two (or more) affiliations. 

However, more important is that if there is only one affiliation: Nothing will change. I guess this is a strong argument. In the worst case (the additional line is not wanted), it takes two seconds in the copyediting process to remove this line.

Reply:
> However, more important is that if there is only one affiliation: Nothing will change.

In the example first author has only one affiliation and there is a new line before the url, ORCID and email. So it seems to me this has changed.

> So I added the additional line to make clear that it is valid for the two affiliations (same for ORCID and URL). Does it make somehow sense to you?

And if you group all in one block, would that not be enough ? It would be pretty clear that the email, orcid and url are for both affiliations.

Reply:
@cderv Ok, I will try this and see how it works out.

Reply:
Resolved, please check.

Reply:
That is awesome ! Thanks!